### PR TITLE
Run macupdater in separate process, instead of thread

### PR DESCRIPTION
**Pull request by dchirikov**

### CODE REVIEW

**Comment by dchirikov** [comment]
Jenkins test this please

**Comment by hi-a** [comment]
Looks good to me

File: bin/lweb [hunk 1]
```diff
@@ -208,7 +220,7 @@ class LWeb(object):
         self.logger.info("Received SIGTERM on PID = {}".format(my_pid))
         tornado.ioloop.IOLoop.instance().stop()
 
-        if self.starter_pid == my_pid:
+        if self.macupdater is not  None:
```
Comment (kostty) [inline bin/lweb]: double space

**Comment by dchirikov** [comment]
Jenkins test this please

**Pull request merged**